### PR TITLE
Add Felinid Laughter

### PR DESCRIPTION
**Pull request by BlitzArde**
## About The Pull Request

This PR allows the Felinid race to use the human laughter sounds when they proc the laugh emote.
## Why It's Good For The Game

Felinids are closely related to humans, and while their voice is distinguishable, the sound of laughter should be the same. It makes happy and cheerful interaction more entertaining when you can actually hear the laugh.
## Changelog
:cl:
sound: Added human laughter to felinids
/:cl:

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -287,14 +287,19 @@
...
-	var/mob/living/carbon/human/human_user = user
-	if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
-		if(human_user.gender == FEMALE)
-			return 'sound/voice/human/womanlaugh.ogg'
-		else
-			return pick('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg')
+    if(!ishuman(user))
+        return
+    var/mob/living/carbon/human/human_user = user
+    if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
```
[the remark this answers is not in the document]
Comment (IsaacExists) [reply code/modules/mob/living/emote.dm:293]: I wanted to do that and then a friend told me they tried and it didn't work, so I trusted their word. I'm likely going to just try that real quick and see if it throws any errors.

[the remark this answers is not in the document]
Comment (IsaacExists) [reply code/modules/mob/living/emote.dm:293]: After testing, I can confirm it does indeed not work, and instead gives other species the laughter too.

**Comment by san7890** [comment]
yeah i think this is a silly oversight

**Comment by san7890** [comment]
Also, for the record, I just approved it after-opening because you're a new contributor. Please note that any future changes to species should probably just be looked over by a maintainer before you open the PR.

**Pull request merged**